packages/http/src/resources/Hotspots.ts

```diff
@@ -24,6 +25,20 @@ export default class Hotspots {
   }
 
   async list(params: ListParams = {}): Promise<ResourceList<Hotspot>> {
+    const { hotspots, cursor } = await this.fetchList(params)
+    const data = hotspots.map((d: HTTPHotspotObject) => new Hotspot(this.client, d))
+    return new ResourceList(data, this.list.bind(this), cursor)
+  }
+
+  async listData(params: ListParams = {}): Promise<ResourceList<HotspotData>> {
```

Comment:
So what happens then if you call HotspotData.activity() or HotspotData.witnesses()? These use the client so they would fail right?

Reply:
you'd be making an intentional choice to terminate your query/chain and just get the data. so you'd probably want to get back a type that reflects that choice maybe? something that doesn't indicate that you could call chaining methods on it? additionally, I could see this `listData` method being hoisted up to a parent class, like `data` in `DataModel`

Reply:
I like the idea of moving the list functions into a base class. I'll investigate our options there.